lib/LoginStrategy.ts

```diff
@@ -239,4 +253,49 @@ export class LoginStrategy extends BaseStrategy {
       return acc
     }, [] as Claim[])
   }
-}
+
+  /**
+   * @description checks if the claimer is authenticated into the DID Document
+   *
+   * @param token
+   * @param payload
+   *
+   * @returns {string} issuer DID or null
+   */
+  async verifyDidClaim(token: string, payload: ITokenPayload): Promise<string> {
+
+    const didDocument = await this.didResolver.read(payload.iss)
+
+    if (this.isAuthorized(token, didDocument))
+      return payload.iss
+    return null 
+  }
+
+  private isAuthorized = (claimedToken: string,didDocument: IDIDDocument) : boolean => {
+
+    //retrieve claim infos from token
+    const claimInfos = jwt.decode(claimedToken) as {[key: string]: any;}
```

Comment:
I think it would be better to install the `@types/jsonwebtoken` package and then you can have an known return type from `jwt.decode` https://www.npmjs.com/package/@types/jsonwebtoken